production/promtail-mixin/dashboards.libsonnet

```diff
@@ -3,24 +3,54 @@ local utils = import 'mixin-utils/utils.libsonnet';
 
 {
   grafanaDashboards+: {
+    local dashboards = self,
+
     'promtail.json':
+      {
+        local cfg = self,
+
+        showMultiCluster:: true,
+        clusterLabel:: 'cluster',
+        clusterMatchers:: if cfg.showMultiCluster then [utils.selector.eq(cfg.clusterLabel, '$cluster')] else [],
+
+        matchers:: [utils.selector.eq('job', '$namespace/$name')],
```

Comment:
What if our `job` label doesn't take this form? I've been using serviceMonitors with prometheus-operator and our jobLabels are unfortunately just the service name currently. I've been meaning to fix it but haven't quite figured out the best method to get `job=$namespace/$name`.

Reply:
I'm a jsonnet newbie, but In hindsight, I'm guessing I would just override `matchers` to do that.